v1.16/Dockerfile

```diff
@@ -13,42 +19,74 @@ RUN apk update \
         ruby ruby-irb ruby-etc \
         tini libmaxminddb geoip \
         snappy \
- && buildDeps=" \
-      make gcc g++ libc-dev \
-      wget bzip2 zlib-dev git linux-headers \
-      automake autoconf libtool build-base \
-      ruby-dev libc6-compat geoip-dev \
-      snappy-dev \
-    " \
  && apk add --no-cache --virtual .build-deps \
         build-base \
         ruby-dev gnupg \
- && apk add $buildDeps \
+ && apk add $BUILD_DEPS \
  && echo 'gem: --no-document' >> /etc/gemrc \
- && gem install bundler && bundle install \
- && apk del .build-deps $buildDeps \
+ && gem install fluentd -v ${FLUENTD_VERSION} \
```

Comment:
isn't this already installed by the base image?

Reply:
[d96e27e](https://github.com/kube-logging/fluentd-images/pull/81/commits/d96e27e0430e03bbd826e047410236d3f6304f5e)